README.md

```diff
@@ -145,15 +145,15 @@ in ``pykeen``.
 
 ### Losses (7)
 
-| Name            | Reference                           | Description                                                                                                                                  |
-|-----------------|-------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
-| bce             | `torch.nn.BCELoss`                  | Creates a criterion that measures the Binary Cross Entropy between the target and the output:                                                |
-| bceaftersigmoid | `pykeen.losses.BCEAfterSigmoidLoss` | A loss function which uses the numerically unstable version of explicit Sigmoid + BCE.                                                       |
-| crossentropy    | `pykeen.losses.CrossEntropyLoss`    | Evaluate cross entropy after softmax output.                                                                                                 |
-| marginranking   | `torch.nn.MarginRankingLoss`        | Creates a criterion that measures the loss given inputs :math:`x1`, :math:`x2`, two 1D mini-batch `Tensors`,                                 |
-| mse             | `torch.nn.MSELoss`                  | Creates a criterion that measures the mean squared error (squared L2 norm) between each element in the input :math:`x` and target :math:`y`. |
-| nssa            | `pykeen.losses.NSSALoss`            | An implementation of the self-adversarial negative sampling loss function proposed by [sun2019]_.                                            |
-| softplus        | `pykeen.losses.SoftplusLoss`        | A loss function for the softplus.                                                                                                            |
+| Name            | Reference                           | Description                                                                                       |
+|-----------------|-------------------------------------|---------------------------------------------------------------------------------------------------|
+| bce             | `pykeen.losses.BCELoss`             | A wrapper around the PyTorch binary cross entropy loss.                                           |
```

Comment:
Not sure what you mean. Should we be linking to the reference PyTorch implementation of some of these, when available?

Reply:
That's already taken care of in the ones that are just wrapping the PyTorch implementations because all classes list their base classes and we have intersphinx linking to the PyTorch documentation.